mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -121,48 +120,57 @@ pub struct TendermintConf {
 
 impl TendermintConf {
     pub fn try_from_json(ticker: &str, conf: &Json) -> MmResult<Self, TendermintInitError> {
-        let avg_block_time = conf["avg_block_time"].as_i64().unwrap_or(0);
+        let avg_blocktime = conf.get("avg_blocktime").ok_or({
+            TendermintInitError {
+                ticker: ticker.to_string(),
+                kind: TendermintInitErrorKind::AvgBlockTimeMissing,
+            }
+        })?;
 
-        // `avg_block_time` can not be less than 1 OR bigger than 255(u8::MAX)
-        if avg_block_time < 1 || avg_block_time > std::u8::MAX as i64 {
-            return MmError::err(TendermintInitError {
+        let avg_blocktime = avg_blocktime.as_i64().ok_or({
+            TendermintInitError {
                 ticker: ticker.to_string(),
-                kind: TendermintInitErrorKind::AvgBlockTimeMissingOrInvalid,
-            });
-        }
+                kind: TendermintInitErrorKind::AvgBlockTimeInvalid,
+            }
+        })?;
+
+        let avg_blocktime = u8::try_from(avg_blocktime).map_err(|_| TendermintInitError {
+            ticker: ticker.to_string(),
+            kind: TendermintInitErrorKind::AvgBlockTimeInvalid,
+        })?;
 
         let derivation_path = json::from_value(conf["derivation_path"].clone()).map_to_mm(|e| TendermintInitError {
             ticker: ticker.to_string(),
             kind: TendermintInitErrorKind::ErrorDeserializingDerivationPath(e.to_string()),
         })?;
 
         Ok(TendermintConf {
-            avg_block_time: avg_block_time as u8,
+            avg_blocktime,
             derivation_path,
         })
     }
 }
 
 pub struct TendermintCoinImpl {
     ticker: String,
-    /// TODO
-    /// Test Vec<String(rpc_urls)> instead of HttpClient and pick
-    /// better one in terms of performance & resource consumption on runtime.
     rpc_clients: Vec<HttpClient>,
+    /// As seconds
+    avg_blocktime: u8,
     /// My address
```

Comment:
Is this comment relevant yet?

Reply:
Yes it is relevant. It's calculated as seconds for cosmos network. I believe it's nice to point this information in data-type.

Reply:
No, I meant `/// My address` comment 😁

Reply:
Oh I see that this comment belongs to `account_id` now